index.html

```diff
@@ -141,8 +194,13 @@ <h3 class="title">Multi-Post Stories Gain+Glory</h3>
     <h2 class="about-me-heading">About Me</h2>
 </div>
 
-    <p class="animate-para"> I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
-    don’t hesitate to contact me.</p>
+    <p> 
+      I can help you build a product, feature or website. Look through some of my work
+      <br>
+       and experience! If you like what you see and have a project you need coded, 
+       <br>
+      don’t hesitate to contact me.
+    </p>
 
     <div class="about-me-button"><button type="button" class="resume-button" >Get My Resume</button></div>
 
```

Comment:
- [ ] I kindly suggest that you follow the Figma design for your project because it's a part of the project requirements. 

| Your Design  | Template Design |
| ------------- | ------------- |
| ![Screenshot from 2022-05-07 23-37-12](https://user-images.githubusercontent.com/52934128/167274291-43e71b0a-b302-4716-b56c-e60f4c4b579b.png)| ![Screenshot from 2022-05-07 23-45-12](https://user-images.githubusercontent.com/52934128/167274272-6a016481-d0f7-4721-8dbb-e11dc4e13c37.png) |